### ISSUE TITLE
Increase player step height

### ISSUE THREAD

**Issue by Latokike** [apace100/origins-fabric#237]
The title says it all. There should be a power type or an entity action to increase/decrease step height.

**Comment by ShareYourHead** [comment]
I agree. I had suggested this on the discord server, so I'll just leave that message here as well.

Could we get a power called origins:step_height? It would allow you to adjust how high a block can be for you to be able to walk straight up it rather than jumping. The default step height for a player is 0.5, since they can walk up slabs, or anything lower. Using the step height power, you could mimic the horse's (and enderman's) behavior where it automatically walks up single blocks, by setting its value to 1. Or you could use it to force the player to jump up the smallest of barriers, such as carpet, by setting its value to 0.
